src/components/common/Modal/index.tsx

```diff
@@ -40,13 +39,17 @@ export const ModalFooter: React.FC<unknown> = ({ children }): JSX.Element => (
   </div>
 );
 
-export const ModalClose = (): JSX.Element => (
-  <button
-    type="button"
-    className="btn-close"
-    data-bs-dismiss="modal"
-    aria-label="Close"
-  />
+interface ModalClose {
+  onClick?: () => void;
+  className?: string[];
+}
+
+export const ModalClose: React.FC<ModalClose> = (
+  { children, onClick = () => {}, className },
+): JSX.Element => (
+  <button type="button" className={!className || !className.length ? 'btn-close' : className.join(' ')} onClick={onClick} data-bs-dismiss="modal" aria-label="Close">
```

Comment:
I would recomment you use `className?: string` and apply it like: `className={className || 'btn-close'}`